src/test/skript/tests/regressions/5682-colors.sk

```diff
@@ -0,0 +1,61 @@
+test "block colors":
```

Comment:
needs to test what happens with beds, signs, banners, non-colourables, etc

Reply:
> needs to test what happens with beds, signs, banners, non-colourables, etc

beds and banners bug out, tried to fix it but I couldn't, make a TODO somewhere about it, do you still want me to add them to the test?

Reply:
Well you need to handle that by not making them bug out. Either properly implement it or don't do anything if it's a bed/banner. So yes you still need to test for them.